cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import socket
+import time
+import os
+
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Waits for database to be ready for connection"  # noqa: VNE003
+
+    def handle(self, *args):
+        port = int(os.environ["POSTGRES_PORT"])  # 5432
+
+        db_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+        print("Waiting for DB connection...")
```

Comment:
Instead of using `print` statements, consider using Django's logging framework. This will give you more control over where these messages are output and how they are formatted.